app/car.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: float) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    @classmethod
+    def create_car(cls, car_info: dict[str, str | float | int]) -> Car:
```

Comment:
The type hint for `car_info` should be `dict[str, Union[str, float]]` instead of `dict[str, str | float | int]` to match the expected types for `brand` and `fuel_consumption`.